docs/notebooks/tutorials/bootstrap_current.ipynb

```diff
@@ -0,0 +1,944 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    eq0.pressure = None
</pre><p>maybe also say here why we make pressure None (bc it will be caluclated from n*T instead)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaControl/DESC/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>